packs/data/age-of-ashes-bestiary.db/saggorak-poltergeist.json

```
@@ -939,39 +939,41 @@
                 },
                 "rules": [
                     {
-                        "domain": "all",
+                        "domain": "attack-roll",
                         "key": "RollOption",
-                        "label": "Telekinetic Storm (Multiple Targets)",
-                        "option": "telekineticStormGroup",
-                        "toggleable": true
+                        "label": "PF2E.NPCAbility.PoltergeistTelekineticStorm.MultiAttackLabel",
+                        "option": "telekineticStormMulti",
```

```suggestion
                        "option": "telekinetic-storm-multi",
```
Let's swap these to kebab-case while we're at it. Roll options should be sluggified this way while actor data should be camelCase.